conf/pawsey_setonix.config

```diff
@@ -0,0 +1,29 @@
+// Pawsey Setonix nf-core configuration profile
+params {
+  config_profile_description = 'Pawsey Setonix HPC profile provided by nf-core/configs'
+  config_profile_contact = 'Sarah Beecroft (@SarahBeecroft), Georgie Samaha (@georgiesamaha)'
+  config_profile_url = 'https://support.pawsey.org.au/documentation/display/US/Setonix+User+Guide'
+}
+
+// Enable use of Singularity to run containers
+singularity {
+  enabled = true
+  autoMounts = true
+  autoCleanUp = true
+}
+
+// Submit up to 256 concurrent jobs (Setonix work partition max)
+executor {
+  queueSize = 1024
+}
+
+// Define process resource limits
+// See: https://support.pawsey.org.au/documentation/pages/viewpage.action?pageId=121479736#RunningJobsonSetonix-Overview
+process {
+  executor = 'slurm'
+  clusterOptions = "--account=${System.getenv('PAWSEY_PROJECT')}"
+  module = 'singularity/3.11.4-slurm'
```

Comment:
Have you tested this? I would be interested to know as my understanding of this directive is that it is an equivalent to conda/docker container definitions themselves (so it's loaded later), and that loading the engine comes in the beforeScript....

But if this works it would be much cleaner!

Reply:
Tested and working. I have found this a little confusing before (I had same understanding as you) but doesn't seem to cause a problem when I've used in place of `beforeScript` in a few applications.